content/en/blog/2020/proxy-cert/index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Provision a certificate and key to your application through files [Experimental]
+description: Provision a certificate and key to your application through files.
+publishdate: 2020-02-20
+attribution: Lei Tang (Google)
+keywords: [certificate,sidecar]
+target_release: 1.5
+---
+
+{{< boilerplate experimental-feature-warning >}}
+
+Istio sidecars can obtain certificates through
+the secret discovery service.
+For an application without a sidecar to get a certificate,
```

Comment:
I think this needs to explain what this is used for. Why would an application not have a sidecar? Why would such an application need a certificate? Explain why Prometheous is a good example, before you describe the example.

You should also clarify how the "application without a sidecar" is different from the application with a sidecar deployed to provision the key and certificate. Both would seem to be "with a sidecar", but the second one is not handling traffic, I presume. I think you need to explain that.

Reply:
The PR has been revised to explain the use case and address the comments.